hack/update-generated-swagger-docs.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Is my following understanding correct: If someone has edited the types.go and forgets to run update docs, then pre-commit will tell him to run update docs; after he runs update docs, pre-commit will fail again and indicates he needs to run update spec.

If my understanding is correct, is it possible to embed update doc & verify doc into update spec & verify spec, so that a user only need to verify & update once?